dest/independentTests/projects.spec.ts

```diff
@@ -0,0 +1,61 @@
+import test, { expect, Page } from "@playwright/test";
+import { projectPageData } from "../../Data/projectsPage.data";
+import { factory } from "../../factory";
+import { v4 as uuid } from 'uuid';
+import homePage from "../../pages/home.page";
+import { profilePage } from "../../pages/profile.page";
+import { projectPage } from "../../pages/project.page";
+import { utils } from "../../Utils/Utils";
+
+test.describe('projects tests', () => {
+    let page: Page;
+    let util: utils;
+    let homePage: homePage;
+    let profilePage: profilePage
+    let projectPage: projectPage;
+
+    let previousNumberOfProject: number;
+
+    test.beforeAll(async ({ browser }) => {
+        var context = await browser.newContext();
+        page = await context.newPage();
+
+        util = factory.initUtils(page);
+        homePage = factory.initHomePage(page);
+        profilePage = factory.initProfilePage(page);
+        projectPage = factory.initProjectsPage(page);
+
+        await homePage.loadHomePage();
+    });
+
+    test.beforeEach(async () => {
+        await profilePage.clickProfilePage();
+        await profilePage.clickProjectsSectionLink();
+        previousNumberOfProject = await profilePage.getNumberOfProjects();
+        await profilePage.clickCreateProject();
+        await projectPage.closePopWindow();
+    })
+
+    test('create a porject test', async () => {
+        let projectName = projectPageData.projectName + uuid();
+        await projectPage.renameProject(projectName);
```

Comment:
where is the assertion part, there should be an "expect" at the end of each test depending on what the test needs to validate

Reply:
in the for each method, it make sure that the count of projects has increased after creating the project and becasue it is a common behavior for each test I put it in there